arm/Microsoft.DesktopVirtualization/applicationgroups/deploy.bicep

```diff
@@ -10,13 +10,13 @@ param location string = resourceGroup().location
   'RemoteApp'
   'Desktop'
 ])
-param appGroupType string
+param type string
 
 @description('Required. Name of the Host Pool to be linked to this Application Group.')
 param hostpoolName string
 
 @description('Optional. The friendly name of the Application Group to be created.')
-param appGroupFriendlyName string = ''
+param friendlyName string = ''
 
 @description('Optional. The description of the Application Group to be created.')
 param appGroupDescription string = ''
```

Comment:
as you updated the others, how about using `description` here too :)

Reply:
description throws and error because it collides with the description key word in bicep :(